ts/ts-diabetes/src/main/java/se/inera/intyg/common/ts_diabetes/v3/rest/TsDiabetesModuleApiV3.java

```diff
@@ -211,4 +217,22 @@ public Certificate getCertificateFromJson(String certificateAsJson,
     public String getJsonFromCertificate(Certificate certificate, String certificateAsJson) throws ModuleException, IOException {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public String getUtlatandeToInternalModelResponse(Utlatande utlatande) throws ModuleException {
+        if (utlatande instanceof TsDiabetesUtlatandeV3) {
+            return toInteralModelResponse((TsDiabetesUtlatandeV3) utlatande);
+        }
+        throw new IllegalArgumentException();
```

Comment:
Include a message with the class of the utlatande. Would be great to see what class the argument had in order to track down the culprit.